tests/transformations/test_clean_imports.py

```diff
@@ -1,141 +0,0 @@
-from libcst.codemod import CodemodTest
-from codemodder.codemods.transformations.clean_imports import CleanImports
-from pathlib import Path
-
-
-class TestCleanImports(CodemodTest):
```

Comment:
It feels weird to delete this entirely since it seems like `CleanImports` is logically distinct from `OrderImports`. I don't want to get bogged down right now but maybe this suggests that there is a slightly better design that better separates the concerns?

Reply:
I thought about that for a bit and, ultimately, the `CleanImports` transform is essentially a combination of `OrderImports` and `RemoveUnusedImports` so the testing is mostly covered by those. 
Honestly it may be the case that we could do away with `CleanImports`. It's not being used anymore.